app/forms/work_form.rb

```diff
@@ -70,15 +70,22 @@ def sync(*)
                if fragment['_destroy'] == '1'
                  contributors.delete(item)
                  return skip!
-               elsif fragment['first_name'].blank?
+               elsif fragment['first_name'].blank? && fragment['full_name'].blank?
                  return skip!
                end
+               if fragment['role_term'].start_with?('person')
```

Comment:
Can you toss a comment here about what's going on? I grok what's going on, but I reckon we'll lose sight of this over time.

Reply:
Good call. I've added it.